{"hierarchy":{"paths":[["doc:\/\/BlueConnect\/documentation\/BlueConnect","doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError"]]},"kind":"symbol","schemaVersion":{"minor":3,"major":0,"patch":0},"metadata":{"modules":[{"name":"BlueConnect"}],"title":"BleCentralManagerProxyError.connectionTimeout","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"connectionTimeout"}],"externalID":"s:11BlueConnect27BleCentralManagerProxyErrorO17connectionTimeoutyA2CmF","symbolKind":"case","role":"symbol","roleHeading":"Case"},"sections":[],"variants":[{"paths":["\/documentation\/blueconnect\/blecentralmanagerproxyerror\/connectiontimeout"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError\/connectionTimeout","interfaceLanguage":"swift"},"primaryContentSections":[{"declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"connectionTimeout"}]}],"kind":"declarations"},{"kind":"content","content":[{"type":"heading","level":2,"text":"Discussion","anchor":"discussion"},{"type":"paragraph","inlineContent":[{"type":"text","text":"This error is thrown when a peripheral connection attempt exceeds the allowed time limit."}]}]}],"abstract":[{"type":"text","text":"Indicates that a timeout occurred during peripheral connection."}],"references":{"doc://BlueConnect/documentation/BlueConnect":{"abstract":[],"role":"collection","type":"topic","identifier":"doc:\/\/BlueConnect\/documentation\/BlueConnect","url":"\/documentation\/blueconnect","kind":"symbol","title":"BlueConnect"},"doc://BlueConnect/documentation/BlueConnect/BleCentralManagerProxyError":{"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"BleCentralManagerProxyError","kind":"identifier"}],"navigatorTitle":[{"text":"BleCentralManagerProxyError","kind":"identifier"}],"type":"topic","identifier":"doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError","role":"symbol","url":"\/documentation\/blueconnect\/blecentralmanagerproxyerror","title":"BleCentralManagerProxyError","kind":"symbol","abstract":[{"text":"An enumeration representing various errors that can occur in the ","type":"text"},{"type":"codeVoice","code":"BleCentralManagerProxy"},{"text":".","type":"text"}]},"doc://BlueConnect/documentation/BlueConnect/BleCentralManagerProxyError/connectionTimeout":{"role":"symbol","kind":"symbol","type":"topic","title":"BleCentralManagerProxyError.connectionTimeout","identifier":"doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError\/connectionTimeout","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"connectionTimeout"}],"abstract":[{"type":"text","text":"Indicates that a timeout occurred during peripheral connection."}],"url":"\/documentation\/blueconnect\/blecentralmanagerproxyerror\/connectiontimeout"}}}